app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt

```diff
@@ -0,0 +1,130 @@
+package com.wire.android.ui.debugscreen
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.ui.common.topappbar.WireTopAppBarTitle
+import com.wire.android.ui.common.topappbar.wireTopAppBarColors
+import com.wire.android.ui.home.conversationslist.common.FolderHeader
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+import com.wire.android.util.startFileShareIntent
+
+@Composable
+fun DebugScreen() {
+    val debugScreenViewModel: DebugScreenViewModel = hiltViewModel()
+    Column {
+        topBar("Debug")
+        list("MLS Data") { debugScreenViewModel.mlsData.map { textRowItem(it) } }
+        list("Logs") { debugLog(debugScreenViewModel) }
+    }
+}
+
+@Composable
+fun topBar(title: String) {
```

Comment:
I'm not sure, but I think we discussed the topic of compose function names, and I feel like we agreed to treat them like these sort of constructors and we usually give them uppercase names 🤔